File: source/MyExpenses/MyExpensesDataGenerator/DataModel/Bonification.cs
namespace MyExpensesDataGenerator
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Expense.Bonification")]
    public partial class Bonification
    {
        public int Id { get; set; }

        public double Amount { get; set; }

        public bool Enabled { get; set; }

        [Required]
        [StringLength(200)]
        public string Description { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime From { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime To { get; set; }

        public int EmployeeId { get; set; }

        public short ExpenseCategoryId { get; set; }

        public virtual Employee Employee { get; set; }

        public virtual Employee Employee1 { get; set; }

        public virtual ExpenseCategory ExpenseCategory { get; set; }
    }
}

File: source/MyExpenses.Web_Experience2/server/model/Expenses/Bonification.js
'use strict';

module.exports = function(sequelize, DataTypes) {
    return sequelize.define('Bonification', {
        id: {
            type: DataTypes.INTEGER,
            allowNull: false,
            primaryKey: true,
            autoIncrement: true 
        },
        amount: {
            type: DataTypes.FLOAT,
            allowNull: false
        },
        enabled: {
            type: 'BIT',
            allowNull: false
        },
        description: {
            type: DataTypes.STRING,
            allowNull: false
        },
        from: {
            type: DataTypes.DATE,
            allowNull: false
        },
        to: {
            type: DataTypes.DATE,
            allowNull: false
        },
        employeeId: {
            type: DataTypes.INTEGER,
            allowNull: false,
            references: {
                model: 'Employee',
                key: 'Id'
            }
        },
        expenseCategoryId: {
            type: DataTypes.INTEGER,
            allowNull: false,
            references: {
                model: 'ExpenseCategory',
                key: 'Id'
            }
        }
    }, {
        timestamps: false,
        tableName: 'Bonification',
        schema: 'Expense'
    });
};
